### DISCUSSION TITLE
How to use a model that is not in the list?

### DISCUSSION THREAD

**Discussion by BlindDeveloper** [a-ghorbani/pocketpal-ai#84] [Q&A] [answered]
Hello, how can I download any model which is not in list in Pocketpal  Android app which downloaded from Play store?

**Comment by jepjoo** [comment, marked as answer]
Download a GGUF file into your phone, open pocket pal, click the "+ local model" button on models page and browse to the downloaded GGUF file and select it.

**Comment by a-ghorbani** [comment]
The app now support hugging face search directly within the app: https://github.com/a-ghorbani/pocketpal-ai/discussions/104
